src/applications/mhv-secure-messaging/tests/e2e/fixtures/thread-response.json

```diff
@@ -159,7 +159,7 @@
                 "subject": "General Inquiry",
                 "body": "testing again",
                 "attachment": null,
-                "sentDate": "20122-10-05T19:28:29.000Z",
+                "sentDate": "2022-10-05T19:28:29.000Z",
```

Comment:
No idea how this got through the CI pipeline. This invalid date with an extra digit in the year was causing an error